components/usage/pkg/db/workspace_instance_usage_test.go

```diff
@@ -16,87 +16,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestCanCreateUsageRecords(t *testing.T) {
```

Comment:
This test was also covered later when it tests the records get updated, there was no reason to keep it.